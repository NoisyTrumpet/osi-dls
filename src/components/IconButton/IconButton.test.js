import React from 'react';
import { render, screen } from '@testing-library/react';
import { toBeInTheDocument } from '@testing-library/jest-dom/matchers';
import { IconButton } from './';

expect.extend({ toBeInTheDocument });

describe('IconButton', () => {
  test('should render', () => {
    const { container } = render(<IconButton icon="achievement" />);
    expect(screen.getByRole('button')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
