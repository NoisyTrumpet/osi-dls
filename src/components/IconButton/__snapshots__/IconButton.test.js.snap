// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton should render 1`] = `
<button
  class="btn-icon btn-icon-simple"
  type="button"
>
  <span
    class="screen-reader-text"
  />
  <span
    aria-hidden="true"
    class="icon icon-achievement icon-16"
    data-testid="icon"
  />
</button>
`;
